gnucash/gnome/gnc-plugin-page-budget.c

```diff
@@ -902,7 +902,7 @@ estimate_budget_helper(GtkTreeModel *model, GtkTreePath *path,
                               GNC_HOW_DENOM_SIGFIGS(priv->sigFigs) | 
                               GNC_HOW_RND_ROUND_HALF_UP);
 
-        if (gnc_reverse_balance(acct))
+        if (gnc_reverse_budget_balance(acct, FALSE))
```

Comment:
Can you help me understand why only two calls to gnc_reverse_budget_balance use FALSE as second parameter (that is, why they emulate pre-4.x behaviour) ? It's difficult to grasp when not having studied the code fully.

Reply:
Or another way to consider this is that FALSE calls can be removed altogether in master.

Reply:
Ok, that makes sense indeed. Conceptually your code looks fine.

I'll try to test-run it one of the coming days.